backend/test/forgotPassword.test.js

```diff
@@ -0,0 +1,125 @@
+const { expect } = require('chai');
+const chaiHttp = require('chai-http');
+const chai = require('chai');
+const Admin = require('../app/models/Admin');
+const server = require('../index');
+
+chai.use(chaiHttp);
+
+before((done) => {
+  // Clearing the test database before testing
+  Admin.remove({}, () => {
```

Comment:
ahh, this caused a mess recently. You are deleting the entire database. You shouldn't do this ever. You are supposed to delete only the resources you create in a test script

Reply:
I personally think this is not a good practice there should be 2 database instances one should be used for `production` and the other for `local development` where our testing could be held on from scratch. And also when I checked the `MONGO_DB_URL` in the `variables.json` file the URL has `/test` written at the end so I inferred that it is just our local DB instance that could be used for testing.

Reply:
Yes it is a test database but you aren't supposed to delete the resources(db entries) which you are not creating during dev/test. Because of you, others need to recreate their entries again and again just because you are testing your code? How is it valid?

Reply:
Oh, makes sense I thought the MongoDB instance is running locally on my system I had no idea that the test DB was common for everyone.

Reply:
Oh, makes sense I thought the MongoDB instance is running locally on my system I had no idea that the test DB was common for everyone. I'll only remove the test admin that I created for testing purposes.